ngmix/bootstrap.py

```diff
@@ -1840,7 +1840,8 @@ def __init__(self, obs,
                                         fracdev_grid['max'],
                                         fracdev_grid['num'])
         else:
-            self.fracdev_tests=linspace(-1.0,1.5,26)
+            # self.fracdev_tests=linspace(-1.0,1.5,26)
+            self.fracdev_tests=linspace(0.0,1.0,26)
```

Comment:
@esheldon here!

Reply:
ok I understand now

Reply:
you can input the fracdev grid parameters, but this is the default.  So better to add something to the config file and then make ngmixer send it on to this bootstrapper

Reply:
agreed we should update the default too though

Reply:
ngmixer just sends everything straight through. It might already work. Let me check.

Reply:
yeah ngmixer just sends it all straight in.

EG, https://github.com/esheldon/ngmixer/blob/master/ngmixer/bootfit.py#L786